### PR TITLE
[ethereum-blockies]: new package

### PR DESCRIPTION
**Pull request by Jack-Works**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by ClassicOldSong** [comment]
LGTM

**Review by SukkaW — APPROVED** [review]

**Review by ClassicOldSong — APPROVED** [review]

**Review by ClassicOldSong — APPROVED** [review]

**Review by magic-akari — APPROVED** [review]

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ethereum-blockies ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ethereum-blockies@latest/))
was missing the following properties:
1. render
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9de8ac383726a4839bf39f817dbafc5402758198
</p>

**Review by limichange — APPROVED** [review]

**Review by lawvs — APPROVED** [review]

**Review by SukkaW — APPROVED** [review]

File: types/ethereum-blockies/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Type definitions for ethereum-blockies 0.1
+// Project: https://github.com/ethereum/blockies (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```
Comment (peterblazejewicz) [inline types/ethereum-blockies/index.d.ts:2]:
```suggestion
// Project: https://github.com/ethereum/blockies
```

**Review by amcasey — CHANGES_REQUESTED** [review]
Please address the comment from @peterblazejewicz.  Otherwise, LGTM

**Review by magic-akari — APPROVED** [review]

**Review by exoticknight — APPROVED** [review]

**Review by ClassicOldSong — APPROVED** [review]

**Review by lawvs — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Review by amcasey — APPROVED** [review]
Thanks!

**Comment by Jack-Works** [comment]
Ready to merge

**Pull request merged**